src/ripple/shamap/SHAMap.h

```diff
@@ -101,13 +113,14 @@ class SHAMap
     SHAMap (
         SHAMapType t,
         Family& f,
-        std::uint32_t seq = 1
```

Comment:
We are _always_ setting it to 1.  This logic is added back here:

https://github.com/ripple/rippled/pull/1647/files/f4d77fff42d1f83b35474966b17203621762ba03#diff-cdf9517dc2cc23f526612bcaf5d27918R33